apps/src/javalab/PlaygroundImage.jsx

```diff
@@ -48,21 +49,27 @@ export default class PlaygroundImage extends React.Component {
   }
 
   render() {
-    const {fileUrl, id, onClick} = this.props;
-    const dynamicStyles = this.getDynamicStyles();
+    const {fileUrl, id, onClick, isClickable} = this.props;
+    let dynamicStyles = this.getDynamicStyles();
+    if (isClickable) {
+      dynamicStyles = {...dynamicStyles, ...styles.clickableImage};
+    }
     return (
       <img
         src={fileUrl}
-        style={{...dynamicStyles, ...styles.imageStyle}}
+        style={{...dynamicStyles, ...styles.image}}
         id={id}
-        onClick={onClick}
+        onClick={isClickable ? onClick : undefined}
       />
     );
   }
 }
 
 const styles = {
-  imageStyle: {
+  image: {
     position: 'absolute'
+  },
+  clickableImage: {
+    cursor: 'pointer'
```

Comment:
Any issues with z-indexing here, i.e. if a non-clickable image is overlaid on a clickable image?

Reply:
If we overlay a non-clickable image on a clickable image, the area with the non-clickable image will have the arrow cursor and won't trigger the click event, which seems like reasonable behavior. If they overlap completely the image below won't be clickable, if there is some overhang of the clickable image below that will be clickable (and have the expected pointer)